python/ray/autoscaler/_private/kuberay/utils.py

```diff
@@ -73,3 +84,24 @@ def parse_quantity(quantity):
 
     exponent = Decimal(exponents[suffix[0]])
     return number * (base**exponent)
+
+
+def tpu_node_selectors_to_type(topology: str, accelerator: str) -> Optional[str]:
```

Comment:
Can you check with Allen to make sure this logic is consistent with the GCE implementation?

Reply:
LGTM:
https://github.com/ray-project/ray/blob/364e65ec9614fa39b4cdf354416a281107003795/python/ray/autoscaler/_private/gcp/config.py#L52-L72

https://github.com/ray-project/ray/blob/364e65ec9614fa39b4cdf354416a281107003795/python/ray/tests/gcp/test_gcp_node_provider.py#L398-L416